src/test/suite/fixtures/recorded/actions/sortEveryItem.yml

```diff
@@ -0,0 +1,48 @@
+languageId: markdown
+command:
+  spokenForm: sort every item
+  version: 3
+  targets:
+    - type: primitive
+      modifiers:
+        - type: everyScope
+          scopeType: {type: collectionItem}
+  usePrePhraseSnapshot: true
+  action: {name: sortTargets}
+initialState:
+  documentContents: 90, 11, 9, 3
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 0}
+  marks: {}
+finalState:
+  documentContents: 3, 9, 11, 90
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 0}
+  thatMark:
```

Comment:
I do wonder if we want to start having specific tests for the `thatMark`.  Even limiting it to actions results in a lot of noise 🤔

Obv out of scope here but just a thought

Reply:
Definitely worth thinking about.